nx-X11/lib/X11/XlibInt.c

```diff
@@ -431,28 +420,57 @@ _XWaitForWritable(
 #endif
 
 #ifndef NX_TRANS_SOCKET
-#  ifdef USE_POLL
+#ifdef USE_POLL
 	    filedes.events = POLLIN;
 	filedes.events |= POLLOUT;
-#  else
+#else
 	FD_SET(dpy->fd, &r_mask);
         FD_SET(dpy->fd, &w_mask);
-#  endif
+#endif
 #endif /* #ifndef NX_TRANS_SOCKET */
 
 	do {
+#ifdef NX_TRANS_SOCKET
```

Comment:
HI Uli, hi Mihai,

On  Fr 26 Aug 2016 06:00:59 CEST, Mihai Moldovan wrote:

>>  #endif /* #ifndef NX_TRANS_SOCKET */
>>
>>  	do {
>> +#ifdef NX_TRANS_SOCKET
>
> NX code reinserted here... (okay, just as a follow-up to the first  
> commit's comment(s).)

without having taken a closer look at the  
"code-removed-code-readded-commits, would it make sense to flatten the  
commit history, so that we don't have commits it this PR that remove  
code and later-on re-add code? Personally, I'd prefer the code base  
being intact after every single commit (of course, we may break stuff  
from time to time, but if it can be avoided, we should IMHO).

Greets,
Mike
-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de